rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java

```diff
@@ -268,6 +268,37 @@
         }
     }
 
+    /**
+     * 
+     * Check if the user has the permission to execute the method passed as argument
+     * 
+     * @param sessionId
+     * @param jobId
+     * @param method
+     * @return true if the user has the permission to execute the java method
+     * @throws NotConnectedRestException
+     * @throws UnknownJobRestException
+     */
+    @Override
+    @GET
+    @Path("job/{jobid}/permission/{method}")
+    @Consumes(value = MediaType.APPLICATION_JSON)
+    @Produces("application/json")
+    public boolean checkJobPermissionMethod(@HeaderParam("sessionid") String sessionId,
+            @PathParam("method") String method, @PathParam("jobid") String jobId)
+            throws NotConnectedRestException, UnknownJobRestException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)